### DISCUSSION TITLE
Why over two years Prism does not release any new version?

### DISCUSSION THREAD

**Discussion by soroshsabz** [PrismLibrary/Prism#2870] [General] [answered]
ITNOA

I am not sure, but as I see in https://www.nuget.org/packages/Prism.Core , Prism does not release any new version in these years, my question is, am I understand correctly? Is prism does not active development in these years? and is prism going to die or not?

thanks

**Comment by soroshsabz** [comment]
related to https://github.com/PrismLibrary/Prism.Templates/issues/55

**Comment by brianlagunas** [comment, marked as answer]
If you look at the repo, you can see it is actively being worked on. Our latest commit being just last week. We have not been able to release for a number of reasons. The first one being we have day jobs that keep us very busy, and we work on this in our free time. We prioritize our personal time over OSS time. Another major reason is because MAUI itself is not ready. We are waiting on a number of issues to be fixed in MAUI before we can release, which we estimate will be with .NET 8.

Also, our GitHub sponsors have had access to all our nightly builds and internal releases all this time. If you would like access to the current packages, you can sponsor either Dan or myself here on GitHub.

**Reply by soroshsabz** [reply]
thanks for response :)

I hope to release new version Prism with a lot new enhancement (with C# 12) as soon as possible based on .NET 8 :)
